sdk/identity/azure-identity/src/main/java/com/azure/identity/ManagedIdentityCredential.java

```diff
@@ -24,9 +24,6 @@
     private final VirtualMachineMsiCredential virtualMachineMSICredential;
     private final ClientLogger logger = new ClientLogger(ManagedIdentityCredential.class);
 
-    // TODO: Migrate to Configuration class in Core (https://github.com/Azure/azure-sdk-for-java/issues/14720)
-    static final String PROPERTY_IDENTITY_ENDPOINT = "IDENTITY_ENDPOINT";
-    static final String PROPERTY_IDENTITY_HEADER = "IDENTITY_HEADER";
```

Comment:
Interesting, so this was in azure-identity earlier. What's the reason for moving this to azure-core? I guess these two are needed only when we use the azure-identity library.